.travis.yml

```diff
@@ -69,6 +69,7 @@ before_script:
 script:
   - java -version
   - bin/scalafmt --test && sbt sbt-crossproject-test/scripted
+  - sbt "^^ 1.0.0-RC2" sbt-crossproject/compile
```

Comment:
Could you make it `sbt-crossproject/publishLocal` so that we make sure that all the artifacts necessary for online publishing will work? In my experience generating documentation can often break while the rest passes, under `-Xfatal-warnings`.